nighthawk/source/common/statistic_impl.cc

```diff
@@ -16,8 +16,8 @@ std::string StatisticImpl::toString() const {
 nighthawk::client::Statistic StatisticImpl::toProto() {
   nighthawk::client::Statistic statistic;
   statistic.set_count(count());
-  statistic.mutable_mean()->set_nanos(std::round(mean()));
-  statistic.mutable_pstdev()->set_nanos(std::round(pstdev()));
+  statistic.mutable_mean()->set_nanos(count() == 0 ? 0 : std::round(mean()));
+  statistic.mutable_pstdev()->set_nanos(count() == 0 ? 0 : std::round(pstdev()));
```

Comment:
Good catch. Can you meaningfully compute std dev if `count() == 1`?

Reply:
Well, as I understand the result of `pstdev()` should be `0` when `count() == 1` holds. For `stdev()` it would be undefined/nan/etc.  We check for that here:

https://github.com/envoyproxy/envoy-perf/blob/453a895285603566f741a1ebf466e1d44f113438/nighthawk/test/statistic_test.cc#L90

Some discussion on the topic:
https://stackoverflow.com/questions/8023878/standard-deviation-of-one-element